packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -274,18 +288,17 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
 
 type UseNftWidgetReturn = NftWidgetProps | null
 export const useNftWidget = (videoId?: string): UseNftWidgetReturn => {
```

Comment:
Seems like almost everything that is in this hook should now just rely on `useNftState`?